model/src/main/java/org/mskcc/smile/model/tempo/MafComplete.java

```diff
@@ -0,0 +1,70 @@
+package org.mskcc.smile.model.tempo;
+
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+
+/**
+ * Represents a MAF Complete entity.
+ * @author qu8n
+ */
+@NodeEntity
+public class MafComplete {
+    @Id @GeneratedValue
+    private Long id;
+    private String date;
+    private String normalPrimaryId;
+    private String status;
+
+    public MafComplete() {}
+
+    /**
+     * Constructor for MafComplete.
+     * @param date
+     * @param normalPrimaryId
+     * @param status
+     */
+    public MafComplete(String date, String normalPrimaryId, String status) {
+        this.date = date;
+        this.normalPrimaryId = normalPrimaryId;
```

Comment:
Note that while the incoming JSON names this property `normal_primaryId`, I'm renaming it using camel case to fit Java's naming convention.